### PR TITLE
Data units of measure with two decimal places.

### PR DESCRIPTION
**Pull request by MadayAlcala**
Hey guys, I changed the byte decimals, please check it out.

### CODE REVIEW

File: src/main/java/com/fundation/logic/common/ByteConvert.java [hunk 1]
```diff
@@ -24,13 +24,13 @@ public static Float bytesConvert(String bytesNumber, String unitConversion) {
...
+                return Math.round(Float.parseFloat(bytesNumber)*100)/100f;
             case "KBytes":
-                return (Float.parseFloat(bytesNumber) / unitBase);
+                return Math.round((Float.parseFloat(bytesNumber) / unitBase)*100)/100f;
             case "MBytes":
-                return ((Float.parseFloat(bytesNumber) / unitBase) / unitBase);
+                return Math.round(((Float.parseFloat(bytesNumber) / unitBase) / unitBase)*100)/100f;
             case "GBytes":
-                return (((Float.parseFloat(bytesNumber) / unitBase) / unitBase)/ unitBase);
+                return Math.round((((Float.parseFloat(bytesNumber) / unitBase) / unitBase)/ unitBase)*100)/100f;
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/common/ByteConvert.java]: put specifics space on /    *

File: src/main/java/com/fundation/logic/common/Validators.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /**
  * Copyright (c) 2019 Jalasoft.
- *
+ * <p>
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/common/Validators.java]: delete < p >

Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/common/Validators.java]: Done.

File: src/main/java/com/fundation/logic/common/ByteConvert.java (hunk 1 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/common/ByteConvert.java]: Done.

File: src/main/java/com/fundation/logic/common/ByteConvert.java [hunk 3]
```diff
@@ -1,6 +1,5 @@
 /**
  * Copyright (c) 2019 Jalasoft.
- *
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/common/ByteConvert.java:3]: put a empty line

**Pull request merged**